src/trackAssessment/enums/statuses.enum.ts

```diff
@@ -0,0 +1,9 @@
+export enum NipunStatus {
```

Comment:
@choxx Can we not add Nipun as status in assessment status? or can we remove nipun status from shiksha adapter and add those in HP?

Reply:
Yes sure. I thought of keeping the statuses enum generic inside a single file; I forgot HP flow has to be a different adapter.
Removed them now. Reference commit: https://github.com/shiksha-platform/backend-v2/pull/127/commits/8731f7c5b9ad65d16db3542c1177c65b6dff69e9

Please check.